docs_sphinx/developer/guidelines/documentation.rst

```diff
@@ -4,9 +4,7 @@ Documentation
 It is very important to maintain documentation. We use the
 `Sphinx documentation generator <http://www.sphinx-doc.org/en/stable/>`__
 tools. The documentation is all hand written. Sphinx source files are stored in the
-``docs_sphinx`` folder (currently: ``dev/brian2/docs_sphinx``). The HTML files
-can be generated via the script ``dev/tools/docs/build_html_brian2.py`` and end
-up in the ``docs`` folder (currently: ``dev/brian2/docs``).
+``docs_sphinx`` folder.                                                      
```

Comment:
The change is correct, but it should keep the references to the ``dev/tools/docs/build_html_brian2.py`` script and the `docs`` folder.